tpm/aziot-tpm-sys/README-TPM.md

```diff
@@ -0,0 +1,118 @@
+
+# Setting up a TPM device with iotedged access
```

Comment:
There is no `iotedged` in this repo.

Reply:
Ah, whoops, this README (and the other one as well) were copied wholesale from the old repo.

That said, I'm not sure how much value these READMEs add over the existing in-repo docs regarding setting up a TPM, and I was thinking of just deleting them entirely. Thoughts?

Reply:
This file `README-TPM.md` has useful info, but also is redundant with a similar page in docs.microsoft.com, so I'd be down for removing it.

The other one has useful info that is not duplicated anywhere, so I would keep that one.